src/Files/MultilingualResources/Files.it-IT.xlf

```diff
@@ -1044,7 +1044,7 @@
         </trans-unit>
         <trans-unit id="PropertyEncodingBitrate" translate="yes" xml:space="preserve">
           <source>Encoding Bitrate</source>
-          <target state="translated">Velocità in bit</target>
+          <target state="translated">Bitrate</target>
```

Comment:
I suggest not changing this translation, at it is the same as the original
![image](https://user-images.githubusercontent.com/11839341/153583029-322037b9-cd2d-4bb0-82ff-8f20a0c6f303.png)
```suggestion
          <target state="translated">Velocità in bit</target>
```